src/core/server/http/http_service.ts

```diff
@@ -102,14 +99,26 @@ export class HttpService implements CoreService<HttpServiceSetup, HttpServiceSta
     }
 
     return {
```

Comment:
__question:__ do we consume this return value anywhere?

Reply:
the core doesn't use it anymore. @toddself are you going to use `isListening` somewhere?